e2e/runner/run_cypress_local.ts

```diff
@@ -36,7 +36,7 @@ const derivedOptions = {
   BUILD_JAR: userOptions.BACKEND_PORT === 4000,
   START_BACKEND: userOptions.BACKEND_PORT === 4000,
   CYPRESS_IS_EMBEDDING_SDK: String(userOptions.TEST_SUITE === "component"),
-  MB_SNOWPLOW_AVAILABLE: userOptions.START_CONTAINERS,
+  MB_SNOWPLOW_AVAILABLE: true,
```

Comment:
Does this even do anything anymore?

Reply:
Yes.
It's apparently needed by the backend. I didn't know that until all Snowplow tests started exploding.